build/publish/RuntimeCoherence.targets

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="RuntimeCoherence"
+          DependsOnTargets="ReadAspNetCoreSharedRuntimeVersionFile;
```

Comment:
I don't know if this is redundant, the following should covered it already. However I am not sure this is a common practice in build script
```xml
 <Target Name="EvaluateRuntimeCoherence"
DependsOnTargets="ReadAspNetCoreSharedRuntimeVersionFile">
```